assignments/objects.js

```diff
@@ -49,7 +91,32 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+	"name": "Susan",
+	"age": 70,
+	"child": {
+		"name": "George",
+		"age": 50,
+		"grandchild":{
+			"name": "Sam",
+			"age": 30,
+			"speak": function(){
+		  		return("Hello, my name is " + this.name);
+		  	}
+		},
+		"speak": function(){
+			return("Hello, my name is " + this.name);
+		}
+	},
+	"speak": function(){
+  		return("Hello, my name is " + this.name);
```

Comment:
You could use a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here:
```javascript
return `Hello, my name is ${this.name}!`;
```